task-08/README.md

```diff
@@ -39,10 +41,10 @@ RIOT provides three shell to interact with the CCN-Lite stack:
 
 * You can use the shell commands above to create some content chunks and send
   out interests for a specific name
-* Create some content for /dagstuhl/m2m/<youraccountname>
-* Send an interest for /dagstuhl/m2m/<otheraccountname>
+* Create some content for /monami/m2m/<youraccountname>
```

Comment:
I would keep this first name component independent of the current conference/event. Otherwise, the next one will change it again from `monami` to something else.